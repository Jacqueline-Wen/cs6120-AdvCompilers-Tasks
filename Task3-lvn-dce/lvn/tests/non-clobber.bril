@main {
  x: int = const 1;
  y: int = id x;
  x: int = add x x;
  print y;
}
