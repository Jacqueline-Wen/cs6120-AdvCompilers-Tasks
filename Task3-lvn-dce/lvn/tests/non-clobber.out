{"functions":[{"instrs":[{"dest":"x_copy0","op":"const","type":"int","value":1},{"args":["x_copy0"],"dest":"y","op":"id","type":"int"},{"dest":"x","op":"const","type":"int","value":2},{"args":["x_copy0"],"op":"print"}],"name":"main"}]}
